From 6f598c6d35c73e9c4e84583ba1ca5c2d92fae989 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:35:43 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(17da287)..lineage-16.0-android-9.0.0_r31(e644222) (DO NOT
 SUBMIT)

Android Build Merger (Role) (5):
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91 am: 52f19a1dd5
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91 am: 52f19a1dd5 am: d339032ec9
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91 am: 52f19a1dd5 am: d339032ec9 am:
   d5bc270755

JP Sugarbroad (4):
 - Support git submodules in version.cc generation.
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91 am: 52f19a1dd5 am: d339032ec9 am:
   d5bc270755 am: d04a70ea7f
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91 am: 52f19a1dd5 am: d339032ec9 am:
   d5bc270755 am: d04a70ea7f am: 23b67dc9ed
 - [automerger] Support git submodules in version.cc generation. am:
   3b166ce40a am: 7f45c35f91 am: 52f19a1dd5 am: d339032ec9 am:
   d5bc270755 am: d04a70ea7f am: 23b67dc9ed am: 181207ea02

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5044688 from d019345012951498cda181caed55908f7fe4f801 to
   pi-qpr2-release

Change-Id: I3eec20cc2f8ccbfdb21f08ded5e0db9f4c2accdc

Change-Id: I5f15afa7075cf7e8557f2936bd2fa0ae4b4e0aa7
---
 Makefile.ckati | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.ckati b/Makefile.ckati
index e4067bb..5bc7c9d 100644
--- a/Makefile.ckati
+++ b/Makefile.ckati
@@ -102,9 +102,9 @@ $(KATI_CXX_TEST_EXES): $(KATI_BIN_PATH)/%: $(KATI_INTERMEDIATES_PATH)/%.o
 	$(KATI_LD) $^ -o $@ $(KATI_LIBS)
 
 # Rule to generate version.cc
-KATI_GIT_DIR := $(shell git -C $(KATI_SRC_PATH) rev-parse --show-toplevel)
+KATI_GIT_DIR := $(shell cd $(KATI_SRC_PATH); realpath `git rev-parse --git-dir`)
 ifneq ($(KATI_GIT_DIR),)
-KATI_VERSION_DEPS := $(KATI_GIT_DIR)/.git/HEAD $(KATI_GIT_DIR)/.git/index
+KATI_VERSION_DEPS := $(KATI_GIT_DIR)/HEAD $(KATI_GIT_DIR)/index
 KATI_VERSION := $(shell git -C $(KATI_GIT_DIR) rev-parse HEAD)
 else
 KATI_VERSION_DEPS :=
-- 
2.17.1

